doc/guides/writing-tests.md

```diff
@@ -18,6 +18,15 @@ Add tests when:
 - Fixing regressions and bugs.
 - Expanding test coverage.
 
+## Test directory structure
+See the test [directory structure overview](https://github.com/nodejs/node/blob/master/test/README.md)
+for an outline of existing test types and their locations.
+When deciding on whether to expand an existing test file or create a new one, 
+consider going through the files related to the subsystem.
+For example, take a look at tests starting with "test-streams" 
+when writing tests for `lib/streams.js`
```

Comment:
The line breaks in this paragraph are actually not visible when viewing this not in raw form. Therefore it would be good to wrap each line around 80 characters.

Reply:
changes made.